.github/workflows/lint.yml

```diff
@@ -27,6 +27,11 @@ jobs:
       - name: 📥 Install deps
         run: yarn --frozen-lockfile
 
+      - name: Disable Eslint GitHub Actions Annotations
+        run: |
+          echo "::remove-matcher owner=eslint-compact::"
+          echo "::remove-matcher owner=eslint-stylish::"
```

Comment:
eslint reporting is set up via `actions/setup-node` https://github.com/actions/setup-node/blob/64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c/src/main.ts